packages/manager/src/features/Volumes/VolumesLandingEmptyState.tsx

```diff
@@ -0,0 +1,166 @@
+    text: 'How to Use Block Storage with Your Linode',
+  },
+  {
+    to: 'https://www.youtube.com/watch?v=8G0cNZZIxNc',
+    text: 'Block Storage Vs Object Storage',
+  },
+  {
+    to: 'https://www.youtube.com/watch?v=Z9jZv_IHO2s',
+    text:
+      'How to use Block Storage to Increase Space on Your Nextcloud Instance',
+  },
+];
+
+const gaCategory = 'Volumes landing page empty';
+const linkGAEventTemplate = {
+  category: gaCategory,
+  action: 'Click:link',
+};
+
+const volumesGuidesMoreLinkText = 'Browse Linode Block Storage documentation';
+
+const guideLinks = (
+  <List>
+    {guidesLinkData.map((linkData) => (
+      <ListItem key={linkData.to}>
+        <Link
+          to={linkData.to}
+          onClick={getLinkOnClick(linkGAEventTemplate, linkData.text)}
+        >
+          {linkData.text}
+        </Link>
+      </ListItem>
+    ))}
+  </List>
+);
+
+const youtubeLinks = (
+  <List>
+    {youtubeLinkData.map((linkData) => (
+      <ListItem key={linkData.to}>
+        <Link
+          onClick={getLinkOnClick(linkGAEventTemplate, linkData.text)}
+          to={linkData.to}
+        >
+          {linkData.text}
+          <ExternalLinkIcon />
+        </Link>
+      </ListItem>
+    ))}
+  </List>
+);
+
+const VolumesLandingEmptyState = () => {
+  const { push } = useHistory();
+
+  return (
+    <StyledPlaceholder
+      title="Volumes"
+      subtitle="NVM block storage service"
+      icon={VolumeIcon}
+      isEntity
+      showTransferDisplay
+      buttonProps={[
+        {
+          onClick: () => {
+            sendEvent({
+              category: gaCategory,
+              action: 'Click:button',
+              label: 'Create Volume',
+            });
+            push('/volumes/create');
+          },
+          children: 'Create Volume',
+        },
+      ]}
+      linksSection={
+        <LinksSection>
+          <LinkSubSection
+            title="Getting Started Guides"
+            icon={<DocsIcon />}
+            MoreLink={(props) => (
+              <Link
+                onClick={getLinkOnClick(
+                  linkGAEventTemplate,
+                  volumesGuidesMoreLinkText
+                )}
+                to="https://www.linode.com/docs/products/storage/block-storage/"
+                {...props}
+              >
+                {volumesGuidesMoreLinkText}
+              </Link>
+            )}
+          >
+            {guideLinks}
+          </LinkSubSection>
+          <LinkSubSection
+            title="Video Playlist"
+            icon={<YoutubeIcon />}
+            external
+            MoreLink={(props) => (
+              <Link
+                onClick={getLinkOnClick(
+                  linkGAEventTemplate,
+                  youtubeMoreLinkLabel
+                )}
+                to="https://www.youtube.com/playlist?list=PLTnRtjQN5ieb4XyvC9OUhp7nxzBENgCxJ"
+                {...props}
+              >
+                Check out our YouTube channel
```

Comment:
Just noticed another small copy inconsistency between this page and the others.

Volumes landing:
![image](https://user-images.githubusercontent.com/114685994/235260425-6cc8fa62-688f-4a7f-8462-10030741f45b.png)


Linode and Database landing:
![image](https://user-images.githubusercontent.com/114685994/235260504-5c25a3c3-da71-4336-9c44-f87739407c10.png)
```suggestion
                View our YouTube channel
```

Reply:
I agree that we should be consistent with this link and Mariah's suggested change